libbpf-rs/src/program.rs

```diff
@@ -772,6 +772,28 @@ impl Program {
         )
     }
 
+    /// Attach this program to a
+    /// [BPF iter](https://www.kernel.org/doc/html/latest/trace/tracepoints.html).
```

Comment:
This link destination does not seem on topic. Is https://www.kernel.org/doc/html/latest/bpf/bpf_iterators.html more fitting?

Reply:
Yes, you're right. I copied/pasted a wrong link.

I will fix it in a few days once after I come back from  my vacation.